### PR TITLE
Add Bootstrap and UI changes.

### PR DESCRIPTION
**Pull request by GunnarPDX**
I added Bootstrap (5) to the application, applied the new SCSS classes to live views, and created a theme file.  

The biggest change was to the tables, I cleaned up the styling/formatting and applied a responsive class to make the tables side scroll instead of smooshing everything vertically.

Other changes included tweaks to the theme colors, improving navs, and minor layout changes.  Also added some info on setting up OAuth to the Docs.

Screen Shot 2020-10-03 at 1 00 15 PM
Screen Shot 2020-10-03 at 1 00 43 PM
Screen Shot 2020-10-03 at 1 00 57 PM
Screen Shot 2020-10-03 at 1 00 28 PM

### CODE REVIEW

**Comment by rupurt** [comment]
Sorry @GunnarPDX, I didn't have notifications on the repo so missed this over the weekend. I'll take a deeper look later this week, but first glance looks great 😎

**Comment by GunnarPDX** [comment]
I'm glad you like it, let me know if you think anything should be changed 😃.  I also added a few SCSS container-breakpoint overrides for wider desktop screen widths.

File: assets/css/app.scss [hunk 1]
```diff
@@ -11,15 +11,18 @@
 /**
  * Setup
  */
-@import "./phoenix";
+// @import "./phoenix";
```
Comment (rupurt) [inline assets/css/app.scss:14]: :scissors:

File: assets/css/app.scss [hunk 2]
```diff
@@ -11,15 +11,18 @@
...
 @import "./custom_reset";
 
 /**
  * Framework
  */
-@import "./buttons";
-@import "./layout";
+// @import "./buttons";
```
Comment (rupurt) [inline assets/css/app.scss:20]: :scissors:

File: assets/css/variables/_colors.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+$dark-grey: #323A46;
+
+// Default text color
+$body-color: #292929;
+
+// Table header color
+$table-header-text: #686868;
+
+// Background color
+$body-bg: #ffffff; //#F2F5F7;
```
Comment (rupurt) [inline assets/css/variables/_colors.scss:18]: :scissors: the old value

File: assets/js/components/BalanceAll.tsx [hunk 4]
```diff
@@ -28,9 +28,9 @@ const BalanceAll: React.FC<Props> = ({ finishTimes, usdBalances, btcBalances, bt
 };
 
 enum Color {
-  Usd = '#02c802',
-  Btc = '#ffb326',
-  BtcUsd = '#6fafff'
+  Usd = '#50ee61', //'#02c802',
```
Comment (rupurt) [inline assets/js/components/BalanceAll.tsx:31]: :scissors: old values

**Review by rupurt — COMMENTED** [review]
This is looking really good. Just a few minor cleanups

**Comment by GunnarPDX** [comment]
I removed the comments, let me know if I should remove the unused files as well like phoenix.scss, I wasn't sure if I should leave them or not.

**Comment by rupurt** [comment]
@GunnarPDX cool. Yeah, let's remove the unused files.

Can you rebase this PR also? I've merge a bunch of dependency update PR's.

**Comment by GunnarPDX** [comment]
Should be gucci now.

**Review by rupurt — APPROVED** [review]
:rocket: :guitar: 

Great stuff

**Pull request merged**